CppCoreGuidelines.md

```diff
@@ -3528,13 +3528,31 @@ There is not a choice when a set of functions are used to do a semantically equi
 
     ???
 
-### <a name="Rf-reference-capture"></a>F.52: Prefer capturing by reference in lambdas that will be used locally, including passed to algorithms
+### <a name="Rf-reference-capture"></a>F.52: When a lambda stays in local scope, prefer capturing by reference, including passed to algorithms
 
 ##### Reason
 
-For efficiency and correctness, you nearly always want to capture by reference when using the lambda locally. This includes when writing or calling parallel algorithms that are local because they join before returning.
+When a lambda is only used locally, capturing by reference is always more efficient.  This includes calling parallel algorithms because they join before returning.
 
-##### Example
+##### Example 1
+
+
+Here, a 'heavy' object (the y-values) is passed to an algorithm in a closure;
+
+	auto interpolated_xs(const std::vector<float>& ys, float start, const float increment, const size_t n)
+	{
+		std::vector<std::pair<float, float>> y_values;
+		std::generate_n(std::back_inserter(x_values), n, [&]
+		{
+			auto y1 = ys[std::floor(start)];
+			auto y2 = ys[std::ceil(start + epsilon)];
+			start += increment;
+			return (y1 + y2) / 2.;
+		});
+		return y_values;
+	}
```

Comment:
Although this is not a requirement here, I think it would make the example easier to understand if it actually compiled. This is also a somewhat strange formula for interpolating points at an arbitrary position.